ngraph_bridge/ie_executable.cc

```diff
@@ -91,59 +92,47 @@ IE_Executable::IE_Executable(shared_ptr<Function> func, string device)
   NGRAPH_VLOG(2) << "Loading IE CNN network to device " << m_device;
 
```

Comment:
Wed can possibly call a InfoSaveInOutIndexMaps() here to save the map

Reply:
For what? There's no need for a map.